e2e/runner/start-sample-app-containers/start-sample-app-containers.ts

```diff
@@ -0,0 +1,67 @@
+import { FAILURE_EXIT_CODE } from "../constants/exit-code";
+import { printBold } from "../cypress-runner-utils";
+import { SAMPLE_APP_SETUP_CONFIGS } from "../sample-apps-shared/constants/sample-app-setup-configs";
+import { fetchApp } from "../sample-apps-shared/helpers/fetch-app";
+import {
+  copyExampleEnvFile,
+  copyLocalEmbeddingSdkPackage,
+  copyLocalMetabaseJar,
+} from "../sample-apps-shared/helpers/prepare-app";
+import { startContainers } from "../sample-apps-shared/helpers/start-containers";
+import type {
+  EmbeddingSdkVersion,
+  SampleAppTestSuiteName,
+} from "../sample-apps-shared/types";
+
+const userOptions = {
+  EMBEDDING_SDK_VERSION: "local",
+  ...process.env,
+} as const;
+
+printBold(`Running Cypress Sample App Tests with options:
+  - EMBEDDING_SDK_VERSION      : ${userOptions.EMBEDDING_SDK_VERSION}
+`);
+
+export async function startSampleAppContainers(
+  testSuite: SampleAppTestSuiteName,
+) {
+  const embeddingSdkVersion =
+    userOptions.EMBEDDING_SDK_VERSION as EmbeddingSdkVersion;
+  const setupConfig = SAMPLE_APP_SETUP_CONFIGS[testSuite];
+  const {
+    appName,
+    branch,
+    "docker-up-command": dockerUpCommand,
+    "docker-down-command": dockerDownCommand,
+    "docker-env-example-path": dockerEnvExamplePath,
+    "docker-env-path": dockerEnvPath,
+    env,
+  } = setupConfig;
+
+  try {
+    const { rootPath } = fetchApp({
+      appName,
+      branch,
+    });
+
+    copyExampleEnvFile({ rootPath, dockerEnvExamplePath, dockerEnvPath });
+
+    copyLocalMetabaseJar(rootPath);
+
+    if (embeddingSdkVersion === "local") {
+      copyLocalEmbeddingSdkPackage(rootPath);
+    }
+
+    await startContainers({
+      cwd: rootPath,
+      env,
+      dockerUpCommand,
+      dockerDownCommand,
```

Comment:
I'm not sure this is consider a config, it seems we don't have different docker up and down command yet, and we put that in `e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js`. Unless we have different commands, how about just inlining the command here? It'd be a lot more straightforward than having to to through different files and try to see where these values are defined.

Of course when we need it we could introduce it back to the config file, but for now YAGNI.

Reply:
I'd keep it as is. It's a configuration value, should not be hardcoded directly in this helper imo